examples/exclude-words/create_wordlist.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import glob
+import json
+from collections import Counter
+from pathlib import Path
+from pdfminer.high_level import extract_text
+
+
+def main(args):
+    file_path = args.file_path
+    minimum_frequency = args.minimum_frequency
+    minimum_length = args.minimum_length
+    allow_numeric = args.allow_numeric
+    words_file = args.words_file
+
+    word_dict = Counter()
+    for file in file_path.iterdir():
+        if file.suffix != '.pdf':
+            continue
```

Comment:
You can use the filetype (which should already be installed) package for even more robust PDF checking. :-)